perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -1455,7 +1450,7 @@ def Run(benchmark_spec):
 
   results = OBJECT_STORAGE_BENCHMARK_DICTIONARY[FLAGS.storage].Run(vms[0],
                                                                    metadata)
-  print results
+  # print results
```

Comment:
We have always printed out the results for ease of debugging, did you remove this (by changing this to a comment) because there are too much information here? (or is it redundant)?

Or is this an accidental change?